browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1473,7 +1478,74 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'type': 'bigtoolitem',
 				'text': _UNO('.uno:TransformDialog'),
 				'command': '.uno:TransformDialog'
-			}
+			},
+			{
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:InsertSparkline'),
+								'command': '.uno:InsertSparkline'
+							},
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:EditSparkline'),
+								'command': '.uno:EditSparkline'
+							},
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:DeleteSparkline'),
+								'command': '.uno:DeleteSparkline'
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:EditSparklineGroup'),
+								'command': '.uno:EditSparklineGroup'
+							},
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:DeleteSparklineGroup'),
+								'command': '.uno:DeleteSparklineGroup'
+							},
+						]
+					}
+				],				
+				'vertical': 'true'
+			},
+			{
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:GroupSparklines'),
+								'command': '.uno:GroupSparklines'
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'type': 'toolitem',
+								'text': _UNO('.uno:UngroupSparklines'),
+								'command': '.uno:UngroupSparklines'
+							}
+						]
+					}
+				],				
+				'vertical': 'true'
+			},
 		];
```

Comment:
Nice, yes to have it in the format tab seems for now the best specially because 1. we do not have better place and 2. it keep the 1 - 1 relation with top menu bar :+1: I would maybe opt to have a menu item here called "Sparklines" so it would occupy less space in the format tab and it would work similar to the classic mode counterpart.

I think this can be done similar to how we did with [Statistics](https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.JSDialogBuilder.js#L137) or [Format Paragraph](https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.JSDialogBuilder.js#L160) etc